pages/api/submitRating.js

```diff
@@ -0,0 +1,25 @@
+import clientPromise from "@/utils/connectMongo";
+
+export default async function handler(req, res) {
+  try {
+    const { id, rating } = req.query;
```

Comment:
maybe it makes sense to use the dynamic route here

Reply:
Alright, yeah, maybe it's something for the last week's optimization! But for now it's not necessary, since there is no dynamic route per card just yet for any method.